lib/core/src/tab/tab.tsx

```diff
@@ -28,14 +28,36 @@ interface TabStyle {
 }
 
 interface Props {
+	/**
+	 * Title and Body of each tab.
+	 */
 	children: Tab[];
+	/**
+	 * Define if tabs padding is removed.
+	 */
 	noPadding?: boolean;
+	/**
+	 * Declare the style of tabs. By default, style is Tabs.style.outset but
+	 * you can change it to Tabs.styles.flat.
+	 */
 	style?: TabStyle;
+	/**
+	 * Define if tabs height is inherited from parents component.
+	 */
```

Comment:
"inherited height" is something not really familiar with many people. Just say that it makes the tabs 100% height of their parent/container